.circleci/config.yml

```diff
@@ -10,6 +10,7 @@ aliases:
         - auto
         - canary
   - &working_directory /opt/git/libra
+  - &image libra/build_environment:circleci-1
```

Comment:
Is the `docker.io` prefix unnecessary? That was removed.

Reply:
it's not.  hoping this will help if multiple dns names exist for dockerhub (they seem to).